docs/community/governance.md

```diff
@@ -73,28 +73,77 @@ long experience with both the project and the larger ecosystem. When the core
 developer community (including the SC members) fails to reach such a consensus
 in a reasonable timeframe, the SC is the entity that resolves the issue.
 
-Members of the steering council also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
+Members of the SC also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
 and are ultimately responsible for managing the napari GitHub account, the [@napari_imaging](https://twitter.com/napari_imaging)
 twitter account, the [napari website](http://napari.org), and other similar napari owned resources.
 
-The steering council is currently fixed in size to three members. This number may be increased in
-the future, but will always be an odd number to ensure a simple majority vote outcome
-is always possible. The initial steering council of napari consists of
-
-- [Juan Nunez-Iglesias](https://github.com/jni)
-- [Loic Royer](https://github.com/royerloic)
-- [Nicholas Sofroniew](https://github.com/sofroniewn)
-
-The SC membership is revisited every January. SC members who do
-not actively engage with the SC duties are expected to resign. New members are
-added by nomination by a core developer. Nominees should have demonstrated
+The SC will be no less than three members and no more than five members,
+with a strong preference for an odd number to ensure a simple majority vote
+outcome is always possible, and a preference for five members to ensure a
+diversity of voices. A deadlocked votes of the SC will be postponed until
+there is an odd number of members and another vote can be held. A majority of the
+SC will not be employed by the same entity. One seat on the SC is reserved
+for a member elected by the [Institutional and Funding Partner Advisory Council](institutional-and-funding-partners),
+as detailed below. This member need not be an existing core developer.
+
+The SC membership, including the Institutional and Funding Partner (IFP) seat, is revisited every January.
+SC members who do not actively engage with the SC duties are expected to resign. New members for 
+vacant spots are added by nomination by a core developer. Nominees should have demonstrated
 long-term, continued commitment to the project and its [mission and values](mission-and-values). A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus. During that time deadlocked votes of the SC will
```

Comment:
It is slightly unclear here whether consensus for admission to the steering council refers to a vote from the core developers/existing steering council and whether this consensus needs to be unanimous. In particular, can the steering council, or any of its members, veto admission of a nominee even when there is consensus among the core developers?

Reply:
My understanding is that it would be a vote of the existing SC. The consensus would not need to be unanimous - but a simple majority of the SC, as is outlined in more detail here https://github.com/napari/napari/blob/main/docs/community/governance.md#decision-making-process.

Reply:
Yes, unless otherwise specified, I would say all decisions are taken by the usual consensus-seeking process, which always ends up in the SC. At any rate I'll second your response to a later comment @sofroniewn — this NAP and PR don't actually change this language, so reviewing it is out of scope for the PR.